### ISSUE TITLE
Review Wiki contents and deactivate

### ISSUE THREAD

**Issue by GaryJones** [Parsely/wp-parsely#177]
There are some development-related pages in the wiki, that could be moved into `.github/CONTRIBUTING.md` or scrapped once #175 is in place. As Parse.ly has it's own public docs website, then general information should be there, and the wiki can be deactivated to reduce user confusion as to where help content is stored.

**Comment by pauarge** [comment]
Content got moved from the Wiki over time and it came up to a couple of pages that were no longer relevant. I have disabled the Wiki page from the GitHub repository.
